VotingApplication/VotingApplication.Web/Scripts/Controllers/DashboardController.js

```diff
@@ -30,15 +33,19 @@
         }
 
         function createPollSuccessCallback(data) {
-            window.location.href = '/#/Manage/' + data.ManageId;
+            navigateToManagePage(data.ManageId);
         }
 
         function getUserPolls() {
             PollService.getUserPolls()
-                .success(function(data) {
+                .success(function (data) {
                     $scope.userPolls = data;
                 });
         }
+
+        function navigateToManagePage(manageId) {
```

Comment:
Perhaps we could pull this out into a service. A separate issue can be raised to convert other pages to use it.